src/main/scala/is/hail/expr/Type.scala

```diff
@@ -563,6 +622,9 @@ abstract class TContainer extends Type {
   // FIXME LCM
   def contentsAlignment: Int = elementType.alignment.max(4)
 
+  override def unsafeOrdering(missingGreatest: Boolean): UnsafeOrdering =
+    TArray(elementType).unsafeOrdering(missingGreatest)
```

Comment:
Just put the definition here instead of on `TArray` (which is a subclass of this class).